central/reportconfigurations/service/service_impl.go

```diff
@@ -203,12 +206,19 @@ func (s *serviceImpl) validateReportConfiguration(ctx context.Context, config *s
 		}
 	}
 
-	_, found, err := s.accessScopeStore.GetAccessScope(ctx, config.GetScopeId())
-	if !found || err != nil {
-		return errors.Wrapf(errox.NotFound, "Access scope %s not found. Error: %s", config.GetScopeId(), err)
+	if features.ObjectCollections.Enabled() {
+		_, found, err := s.collectionDatastore.Get(ctx, config.GetScopeId())
+		if !found || err != nil {
+			return errors.Wrapf(errox.NotFound, "Collection %s not found. Error: %s", config.GetScopeId(), err)
```

Comment:
If the latter, I urge the collections teams to look into it with PM since if its just collection ... it could be a collection of ... rocks! Or marbles! Or artwork.  :-)

Reply:
That's right, keeping this open for now to get answers on whether resource collections is good enough or we need a different name for would be better.

Reply:
Follow the name used in the REST API.

Reply:
The REST API uses the name "Collections".